assets/mongodb_queries/data_qc/applicable_class_was_generated_by_activity.js

```diff
@@ -0,0 +1,36 @@
+var applicableClasses = ["data_object_set", ] // what other sets go here?
+
+var results = [];
+
+applicableClasses.forEach(function(applicableClass) {
+    var pipeline = [
+        {
+            $unwind: "$was_generated_by"
+        },
+        {
+            $lookup: {
+                from: "activity_set",
+                localField: "was_informed_by",
+                foreignField: "id",
+                as: "was_generated_by_docs"
```

Comment:
This is confusing and mixes was_informed_by and was_generated_by.

Reply:
local field should be 'was_generated_by'.
OmicsProcessing is also valid for a value for was_generated, I don't think that is accounted for with 'activity_set'

Reply:
The documentation for `Class:DataObject` https://microbiomedata.github.io/nmdc-schema/DataObject/
has it as 0..1 with Range `Activity`

Reply:
`was_generated_by` slot also shows Range: `Activity`

Reply:
Fixed above issue and added check for both Activity and OmicsProcessing